content/en/docs/getting-started/_index.md

```diff
@@ -15,49 +15,51 @@ description: >
 
-Once you have the `operator-sdk` binary installed, you can easily install OLM on your cluster by running `operator-sdk olm install`. 
-
-```bash 
-$ operator-sdk olm install 
-INFO[0000] Fetching CRDs for version "latest"           
-INFO[0000] Fetching resources for resolved version "latest" 
-I0428 10:07:00.985939 3850425 request.go:655] Throttling request took 1.046148019s, request: GET:https://127.0.0.1:45457/apis/storage.k8s.io/v1beta1?timeout=32s
-INFO[0008] Creating CRDs and resources                  
-INFO[0008]   Creating CustomResourceDefinition "catalogsources.operators.coreos.com" 
-INFO[0008]   Creating CustomResourceDefinition "clusterserviceversions.operators.coreos.com" 
-INFO[0008]   Creating CustomResourceDefinition "installplans.operators.coreos.com" 
-INFO[0008]   Creating CustomResourceDefinition "operatorconditions.operators.coreos.com" 
-INFO[0008]   Creating CustomResourceDefinition "operatorgroups.operators.coreos.com" 
-INFO[0008]   Creating CustomResourceDefinition "operators.operators.coreos.com" 
-INFO[0008]   Creating CustomResourceDefinition "subscriptions.operators.coreos.com" 
-INFO[0008]   Creating Namespace "olm"                   
-INFO[0008]   Creating Namespace "operators"             
-INFO[0008]   Creating ServiceAccount "olm/olm-operator-serviceaccount" 
-INFO[0008]   Creating ClusterRole "system:controller:operator-lifecycle-manager" 
-INFO[0008]   Creating ClusterRoleBinding "olm-operator-binding-olm" 
-INFO[0008]   Creating Deployment "olm/olm-operator"     
-INFO[0008]   Creating Deployment "olm/catalog-operator" 
-INFO[0008]   Creating ClusterRole "aggregate-olm-edit"  
-INFO[0008]   Creating ClusterRole "aggregate-olm-view"  
-INFO[0008]   Creating OperatorGroup "operators/global-operators" 
-INFO[0010]   Creating OperatorGroup "olm/olm-operators" 
-INFO[0010]   Creating ClusterServiceVersion "olm/packageserver" 
-INFO[0012]   Creating CatalogSource "olm/operatorhubio-catalog" 
-INFO[0012] Waiting for deployment/olm-operator rollout to complete 
-INFO[0012]   Waiting for Deployment "olm/olm-operator" to rollout: 0 of 1 updated replicas are available 
-INFO[0026]   Deployment "olm/olm-operator" successfully rolled out 
-INFO[0026] Waiting for deployment/catalog-operator rollout to complete 
-INFO[0026]   Waiting for Deployment "olm/catalog-operator" to rollout: 0 of 1 updated replicas are available 
-INFO[0031]   Deployment "olm/catalog-operator" successfully rolled out 
-INFO[0031] Waiting for deployment/packageserver rollout to complete 
-INFO[0031]   Waiting for Deployment "olm/packageserver" to rollout: 0 of 2 updated replicas are available 
-INFO[0037]   Deployment "olm/packageserver" successfully rolled out 
-INFO[0037] Successfully installed OLM version "latest"  
+Once you have the `operator-sdk` binary installed, you can easily install OLM on your cluster by running `operator-sdk olm install`.
```

Comment:
```suggestion
After you have the `operator-sdk` binary installed, you can install OLM on your cluster by running `operator-sdk olm install`.
```
Two small nits, and perhaps out of scope since this sentence is part of the original content.

Reply:
Updated, thanks.